python-sdk/tests_integration/sql/operators/test_drop_table.py

```diff
@@ -133,7 +133,7 @@ def validate_table_exists(table: Table):
         },
         {
             "database": Database.DUCKDB,
-            "table": Table(conn_id="redshift_conn"),
```

Comment:
@rajaths010494 this should have failed earlier, right?

Reply:
Yes, it should have ideally failed.